### PR TITLE
feat: standalone modal support

### PR DESCRIPTION
**Pull request by Seavenly**

### CODE REVIEW

File: src/components/modal/lib/utils.js [hunk 1]
```diff
@@ -28,16 +28,28 @@ export const getContent = memoizeOnProps(
...
         arrayIncludes(
-            ['EZP:ANY:EQZ', 'EZP:ANY:GTZ', 'PALA:MULTI:EQZ', 'PALA:MULTI:GTZ', 'PALA:SINGLE:EQZ', 'PALA:SINGLE:GTZ'],
+            [
+                'EZP:ANY:EQZ',
+                'EZP:ANY:GTZ',
+                'PALA:MULTI:EQZ',
+                'PALA:MULTI:GTZ',
+                'PALA:SINGLE:EQZ',
+                'PALA:SINGLE:GTZ',
+                'EZP'
```
Comment (merlinstardust) [inline src/components/modal/lib/utils.js]: Small thing but can EZP be put first so all the offers starting with EZP are together?

**Review by merlinstardust — APPROVED** [review]
Existing code and tests look good mostly except for the one small change I had. Plus I had a few thoughts beyond that

1. Can you add a functional test for this?
2. Since this feature is called standalone modal, I feel like we should rename the `demo/standalone.html` file to something else to prevent confusion.
3. Maybe this should get its own demo page like `demo/standalone_modal.html`? Thoughts?

**Review by merlinstardust — CHANGES_REQUESTED** [review]
Accidentally hit approve on the last one, so comments are the same, but clicking request this time.

File: src/controllers/message/setup.js [hunk 2]
```diff
@@ -7,27 +7,29 @@ export default function setup() {
     // Populate global config options
     const script = getScript();
     if (script) {
-        const { merchantid, ...inlineScriptOptions } = getInlineOptions(script);
+        const { ...inlineScriptOptions } = getInlineOptions(script);
```
Comment (gillycheesesteak) [inline src/controllers/message/setup.js]: Can this be changed to `const inlineScriptOptions = ...`

**Review by gillycheesesteak — APPROVED** [review]

**Pull request merged**